app/controllers/samples_controller.rb

```diff
@@ -762,7 +762,8 @@ def create
       project_name = params.delete(:project_name)
       project = Project.find_by(name: project_name)
       unless project
-        project = Project.create(name: project_name)
+        # Web flow would already create the project so this block is for the CLI
+        project = Project.create(name: project_name, metadata_fields: MetadataField.where(is_default: 1))
```

Comment:
Could we add this as a `before_create` hook in the project model instead?

Reply:
👍 moving this to my other PR